README.adoc

```diff
@@ -18,7 +18,7 @@ All three builds (Ant, Gradle, Maven) will automatically download all required d
 - Maven use `mvnw` wrapper
 - Gradle use `gradlew` wrapper
 
-NOTE: This example shows the usage of Spock 2.0, which uses the JUnit Platform. If you want to see how to get Spock 1.x with JUnit 4 up and running see the https://github.com/spockframework/spock-example/tree/spock-1.x[Spock-1.x] Branch.
+NOTE: This example shows the usage of Spock 2.0, which uses the JUnit 5 platform. If you want to see how to get Spock 1.x with JUnit 4 up and running, see the https://github.com/spockframework/spock-example/tree/spock-1.x[Spock-1.x] Branch.
```

Comment:
Sorry to nitpick but this is wrong, as you can see here https://junit.org/junit5/docs/current/user-guide/#overview-what-is-junit-5